chsdi/templates/htmlpopup/shop_product_group.mako

```diff
@@ -0,0 +1,27 @@
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c, lang)">
+
+<%
+   from chsdi.lib.helpers import format_scale, format_price
```

Comment:
I think you don't have to import functions from helpers, they are in the `base.mako`
You can simply do `h.format_scale()` and `h.format_price()`